tests/integration/api_v3/http/generators.test.js

```diff
@@ -61,7 +62,7 @@ describe('Generators API', () => {
 			expect(response.data.length).toBeGreaterThanOrEqual(1);
 			expect(response.data.length).toBeLessThanOrEqual(103);
 
-			// TODO: Verify and fix
+			// // TODO: Verify and fix
```

Comment:
Could you please verify if this works? If so, let's enable it 👍 
```suggestion
			// TODO: Verify and fix
```